packages/edit-site/src/components/editor/index.js

```diff
@@ -92,6 +97,9 @@ function Editor( { settings: _settings } ) {
 		<>
 			<EditorStyles styles={ settings.styles } />
 			<FullscreenMode isActive={ isFullscreenActive } />
+			<MainDashboardButton.Slot>
```

Comment:
I tried testing this with instructions from https://github.com/WordPress/gutenberg/pull/22323 to verify if extensibility of the close button, and it doesn't seem to work anymore. The default button is not replaced.

I'm not sure what's causing this, and whether it's somehow related to moving this outside of editor skeleton. FWIW I don't think that it should be moved out of this area. With this setup it seems that everyone who's aiming to provide a custom button, will also have to provide styles to position it properly on top of the header, and I think that will be prone to breakage over time.